samples/GuiSamples.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
These files shouldn't be part of this PR, they are part of https://github.com/terrafx/terrafx.interop.windows/pull/52